test/functionalTest/cases/0876_entity_dates/get_entity_dates.test

```diff
@@ -57,14 +57,14 @@ echo
 
 echo "02a. Get entity with dates"
 echo "=========================="
-orionCurl --url /v2/entities/E?options=dateCreated,dateModified
```

Comment:
The old copy of this .test has been moved to cases/0000_deprecated_functionality (or something like that)

NTC